protocol/messenger_communities.go

```diff
@@ -532,6 +535,87 @@ func (m *Messenger) updateCommunitiesActiveMembersPeriodically() {
 	}()
 }
 
+func (m *Messenger) HandleCommunityGrantUpdated(state *ReceivedMessageState, update *protobuf.CommunityGrantUpdated, statusMessage *v1protocol.StatusMessage) error {
+	community, err := m.communitiesManager.GetByID(update.CommunityId)
+	if err != nil {
+		return err
+	}
+
+	if community == nil {
+		return communities.ErrOrgNotFound
+	}
```

Comment:
This is already handled inside `GetByID`:
https://github.com/status-im/status-go/blob/aa73a0512c32168f6401d29c7beb7bef78be5158/protocol/communities/manager.go#L3360-L3362

Reply:
fixed